### PR TITLE
chore: use db.system constants from semantic-conventions package

### PR DESCRIPTION
**Pull request by trask**
## Which problem is this PR solving?

- DbSystemValues constants were used in some places but not in others

## Short description of the changes

- use DbSystemValues everywhere for `db.system` attribute values

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts [hunk 1]
```diff
@@ -44,7 +47,7 @@ export class MySQLInstrumentation extends InstrumentationBase<
 > {
   static readonly COMPONENT = 'mysql';
```
Comment (blumamir) [inline plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts:48]: I think it's not used anywhere else. Can be removed

**Review by blumamir — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-redis/src/utils.ts [hunk 2]
```diff
@@ -102,7 +105,7 @@ export const getTracedInternalSendCommand = (
       {
         kind: SpanKind.CLIENT,
         attributes: {
-          [SemanticAttributes.DB_SYSTEM]: RedisInstrumentation.COMPONENT,
+          [SemanticAttributes.DB_SYSTEM]: DbSystemValues.REDIS,
```
Comment (Flarna) [inline plugins/node/opentelemetry-instrumentation-redis/src/utils.ts:108]: is `RedisInstrumentation.COMPONENT` still used anywhere?  if not it should be removed.

**Review by Flarna — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-redis/src/utils.ts (hunk 2 above)
Comment (trask) [reply plugins/node/opentelemetry-instrumentation-redis/src/utils.ts:108]: this one is still used

**Comment by dyladan** [comment]
@trask just confirming `chore`. Our automation uses the title of the PR to automatically determine the semver version change and `chore` does not cause a release because there should be no change in behavior. It seems to me thats fine here, but I want to confirm this is what you intended.

**Comment by trask** [comment]
> @trask just confirming `chore`. Our automation uses the title of the PR to automatically determine the semver version change and `chore` does not cause a release because there should be no change in behavior. It seems to me thats fine here, but I want to confirm this is what you intended.

ya, there should be no change in behavior, the code was already using the correct values, just not pulling them from the semantic-conventions package (whether I _intentionally_ used `chore` correctly is a different question 😂 so thx for checking)

**Pull request merged**